content/en/docs/reference/scheduling/config.md

```diff
@@ -58,6 +58,9 @@ extension points:
    Policy and are used to filter out nodes that can not run the Pod. Filters
    are called in the configured order. A pod is marked as unschedulable if no
    nodes pass all the filters.
+1. `PostFilter`: These plugins are called in their configured order when no
+   feasible nodes were found for the pod. If any postFilter plugin marks a node
+   as `Schedulable`, the remaining plugins are not called.
```

Comment:
Prefer not to emphasize/backquote "Schedulable" as it's just an internal state:

```suggestion
   schedulable, the remaining plugins are not called.
```

Reply:
Done

Reply:
Maybe put _schedulable_ in italics to reflect that it is a specific term with meaning to the code.

Reply:
Done